policies-and-procedures/jellyfin-constitution.md

```diff
@@ -0,0 +1,156 @@
+<!-- markdownlint-disable -->
```

Comment:
To only disable the duplicate headings warning:
```suggestion
<!-- markdownlint-disable MD024 -->
```

Or, even more specific, put this somewhere in the file (could also be at the end):
```
<!-- markdownlint-configure-file { "MD024": { "allow_different_nesting": true } } -->
```

Reply:
I just disabled them all because meh. It complains both about duplicate headers and numbering being explicit.